lighthouse-core/audits/resource-summary.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * @license Copyright 2016 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit');
+const NetworkRecords = require('../computed/network-records.js');
+const i18n = require('../lib/i18n/i18n.js');
+const MainResource = require('../computed/main-resource.js');
+const URL = require('../lib/url-shim');
+const Util = require('../report/html/renderer/util.js');
+
+const UIStrings = {
+  /** Imperative title of a Lighthouse audit that tells the user to minimize the size and quantity of resources used to load the page. */
+  title: 'Keep request counts and file sizes small',
+  /** Description of a Lighthouse audit that tells the user that they can setup a budgets for the quantity and size of page resources. No character length limits. */
+  description: 'To set budgets for the quantity and size of page resources,' +
+    ' add a budgets.json file.',
+  /** [ICU Syntax] Label for an audit identifying the number of requests and kilobytes used to load the page. */
+  displayValue: `{requestCount, plural, =1 {1 request} other {# requests}} • { formattedByteCount}`,
+  /** Label for the combination of all resources on the page. */
+  totalResourceType: 'Total',
+  /** Label for the 'Document' resource type. */
+  documentResourceType: 'Document',
+  /** Label for the 'script' resource type. */
+  scriptResourceType: 'Script',
+  /** Label for the 'stylesheet' resource type. */
+  stylesheetResourceType: 'Stylesheet',
+  /** Label for the 'image' resource type. */
+  imageResourceType: 'Image',
+  /** Label for the 'media' resource type. */
+  mediaResourceType: 'Media',
+  /** Label for the 'font' resource type. */
+  fontResourceTyoe: 'Font',
+  /** Label for the 'other' resource type. */
+  otherResourceType: 'Other',
+  /** Label for the 'third-party' resource type. */
+  thirdPartyResourceType: 'Third-party',
+};
+
+const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
+
+class ResourceSummary extends Audit {
+  /**
+       * @return {LH.Audit.Meta}
+       */
+  static get meta() {
+    return {
+      id: 'resource-summary',
+      title: str_(UIStrings.title),
+      description: str_(UIStrings.description),
+      scoreDisplayMode: Audit.SCORING_MODES.INFORMATIVE,
```

Comment:
Do we only want this to only show up if it exceeds the default budget thresholds or how will that interplay with the custom budget audit that's separate?

Right now, it will show up as a failure on every page even if they have only 1 request weighing in at 1KB :)

Reply:
My understand was that this audit should always be displayed but it shouldn't be considered a failure. Is there a precedent for that type of behavior?

Reply:
Oh summoning @hwikyounglee and @yuinchien then!

Right now an informative audit either has something to tell you, in which case it shows up with the red failure icon, or it's marked as not applicable. This was based on the idea that if an informative audit doesn't really want you to take any action on the information, it should be hidden so that the user focuses on the more important items in the report.

It sounds like we want to change this paradigm here and go back to having an informative audit display state?

Reply:
Is there a screenshot we can look at?

@yuinchien's design has a gray circle. Would it work? 
<img width="783" alt="graycircle" src="https://user-images.githubusercontent.com/7561395/56875662-22037000-69f7-11e9-8012-d1ad30c3cd78.png">

Reply:
Ah, yes so we do want to have an informative audit display state! @Hoten FYI I don't think this is currently tracked in #8185.

This is what it looks like at the moment @hwikyounglee (for future reference, the "View Deployment" links on this PR will take you to the report as it looks with a given PR applied 😃 Paul added this awesome workflow improvement very recently 🎉 )

![image](https://user-images.githubusercontent.com/2301202/56902491-06807f80-6a60-11e9-8eac-c4bb338af9b4.png)

Reply:
Correct, we do want to have an informative audit display state per [this early LW discussion](https://github.com/GoogleChrome/lighthouse/pull/7176#issuecomment-470838225), **colored in gray**.  

So with the new [visual system](https://yuinchien.github.io/sandbox/lighthouse/) applied,
[ ] the row should be ordered as the last within Diagnostics list
[ ] the icon should be **gray circle (not red triangle)** 
[ ] the values on the far right should be in **gray (not red)**.

SGTY @patrickhulce @khempenius @yuinchien?

Reply:
Love it @hwikyounglee! 👍

Reply:
And just to double-check: the grey circle symbol should be applied to all informative audits, right? @hwikyounglee

Reply:
@khempenius correct, all informative audits should have the gray circle symbol and data values in gray as well. Thanks!